pkg/reconciler/tenancy/clusterworkspacetype/clusterworkspacetype_controller.go

```diff
@@ -55,13 +56,17 @@ func NewController(
 	queue := workqueue.NewNamedRateLimitingQueue(workqueue.DefaultControllerRateLimiter(), controllerName)
 
 	clusterWorkspaceShardLister := clusterWorkspaceShardInformer.Lister()
+	clusterWorkspaceTypeLister := clusterWorkspaceTypeInformer.Lister()
```

Comment:
(taste) useless temporary var without value alarm 🚨

Reply:
It's used in a couple of spots and follows the pattern from the other lister... /shrug